ot3_state_manager/pyproject.toml

```diff
@@ -28,6 +30,7 @@ mdformat_footnote = "^0.1.1"
 # Expects the monorepo (opentrons) to be a sibling of this repo
 opentrons-hardware = { path = "../../opentrons/hardware" }
 opentrons = { path = "../../opentrons/api" }
+header_generation_utils = { path = "../header_generation_utils"}
```

Comment:
this dep, and probably `opentrons` and `opentrons-hardware`, want to be [`editable = True`](https://python-poetry.org/docs/cli/#add) if possible

Reply:
https://github.com/Opentrons/ot3-firmware/pull/435/commits/2effe2974c6e39769347d3cf149a189484f6d776

FYI, it is `develop=true`